### PR TITLE
Update french.txt

### PR DESCRIPTION
**Pull request by Sehyn**
Added translation for some untranslated strings.

Fixed a translation mistake -> "Reconnecte dans %s is Reconnexion dans %s"

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/french.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@
 #	T tee & NRK & Chairn	2022-03-23 20:52:52
 #	Nouaa			2022-05-14 23:00:00
 #	Chairn			2022-06-16 19:00:00
+# Sucre       2022-07-19 20:36:00
```
Comment (dannyhpy) [inline data/languages/french.txt:25]:
```diff
- # Sucre       2022-07-19 20:36:00
+ #	Sucre			2022-07-19 20:36:00
```

File: data/languages/french.txt [hunk 2]
```diff
@@ -1515,10 +1517,10 @@ Loading ghost files
 == 
 
 Loading skin files
-== 
+== Chargement des fichiers de skins
 
 Appearance
-== 
+== Appparance
```
Comment (dannyhpy) [inline data/languages/french.txt:1523]:
```diff
- == Appparance
+ == Apparence
```

File: data/languages/french.txt [hunk 3]
```diff
@@ -1536,16 +1538,16 @@ Show client IDs in scoreboard
...
 Show jumps indicator
-== 
+== Montrer l'indicateur de saut
 
 Show hook strength indicator
 == 
 
 Hook collision line
-== 
+== Montrer la collision du grapin
```
Comment (dannyhpy) [inline data/languages/french.txt:1550]:
```diff
- == Montrer la collision du grapin
+ == Ligne de collision du grapin
```

File: data/languages/french.txt [hunk 4]
```diff
@@ -1536,16 +1538,16 @@ Show client IDs in scoreboard
 == 
 
 Show DDRace HUD
-== 
+== Montrer l'UI DDrace
```
Comment (dannyhpy) [inline data/languages/french.txt:1541]: DDRace*

**Review by dannyhpy — CHANGES_REQUESTED** [review]

File: data/languages/french.txt (hunk 2 above)
Comment (Chairn) [inline data/languages/french.txt:1523]: Apparence

File: data/languages/french.txt (hunk 3 above)
Comment (Sehyn) [reply data/languages/french.txt:1550]: I'm not sure about that one cause, in another line it actually talk about hook line.

File: data/languages/french.txt (hunk 4 above)
Comment (Sehyn) [reply data/languages/french.txt:1541]: Absolutely my mistake.

File: data/languages/french.txt [hunk 5]
```diff
@@ -1536,16 +1538,16 @@ Show client IDs in scoreboard
...
 
 Show jumps indicator
-== 
+== Montrer l'indicateur de saut
```
[the remark this answers is not in the document]
Comment (Sehyn) [reply data/languages/french.txt:1544]: To be confirmed if it's pluriel or not.

**Comment by def-** [comment]
@Sehyn can you make the fixes and update this PR?

**Comment by NouaaTW** [comment]
Looks good to me

File: data/languages/french.txt [hunk 6]
```diff
@@ -1527,25 +1529,25 @@ Hook Collisions
 == 
 
 Show health, shields and ammo
-== 
+== Monter la vie, bouclier et munitions
```
Comment (dannyhpy) [inline data/languages/french.txt:1532]:
```diff
+ == Montrer la vie, le bouclier et les munitions
```

**Review by dannyhpy — CHANGES_REQUESTED** [review]

File: data/languages/french.txt [hunk 7]
```diff
@@ -1527,25 +1529,25 @@ Hook Collisions
...
 
 DDRace HUD
-== 
+== HUD DDrace
```
Comment (dannyhpy) [inline data/languages/french.txt:1535]: Il y a une raison particulière à l'écrire "DDrace" au lieu de "DDRace" comme dans la chaîne de caractères initiale ?

Comment (Sehyn) [reply data/languages/french.txt:1535]: Petit problème de caps lock, a modifier du coup.

Si jamais la question de pourquoi HUD DDRace avant et pas DDRace HUD, on dit  l'interface DE quelque chose et non quelque chose de l'interface ou quelque chose interface

**Review by dannyhpy — APPROVED** [review]

File: data/languages/french.txt (hunk 3 above)
Comment (blueskyGH) [reply data/languages/french.txt]: @Sehyn in this case its just talking about the line itself, not about showing it 😄

**Comment by def-** [comment]
bors r+
Thanks

**Pull request merged**